src/prif_m.f90

```diff
@@ -0,0 +1,11 @@
+! Copyright (c), The Regents of the University of California
+! Terms of use are as specified in LICENSE.txt
+module prif_m
+  use program_termination_m, only : prif_stop, prif_error_stop
+  use image_enumeration_m, only : prif_this_image, prif_num_images
+  use collective_subroutines_m, only : prif_co_sum, prif_co_max, prif_co_min, prif_co_reduce, prif_co_broadcast
+  use caffeinate_decaffeinate_m, only : prif_caffeinate, prif_decaffeinate
```

Comment:
There is no specified `prif_caffeinate` nor `prif_decaffeinate`. There is only `prif_init`